HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -368,16 +368,16 @@ method:: asAudioRateInput
 Converts this into an audiorate input.
 
 method:: asTimeString
-Produces a time string in the format code::ddd:hh:mm:ss.sss::, interpreting the receiver as time in
-seconds. See link::Classes/String#-asSecs:: for the inverse function.
+Produces a time string in the clock format inspired by ISO 8601 time interval display (truncated representation) code::ddd:hh:mm:ss.sss::, interpreting the receiver as time in seconds. See link::Classes/String#-asSecs:: for the inverse function.
 argument:: precision
-accuracy of the millisecond format; the string will always be formatted with 3 decimal places for
-milliseconds. Minimum value: 0.001.
+accuracy of the fraction of seconds; since the number of decimal places is also an argument, minimum value of code::precision:: is code::10.pow(decimalPlaces.neg)::, i.e. code::0.001:: for code::decimalPlaces = 3::
 argument::maxDays
 maximum number of days
 argument::dropDaysIfPossible
 a link::Classes/Boolean::. If set to code::true::, and the number of days in the formatted string
-would be 0, that section is instead ommitted and the string is formatted as code::hh:mm:ss.sss::.
+would be 0, that section is instead ommitted and the string is formatted as code::hh:mm:ss.sss::
```

Comment:
since we're editing this -- can you fix the typo? ommitted->omitted. and maybe just leave off the "is formatted as hh:mm:ss.sss" since that isn't technically true anymore.